plugin/builtin/imagetagtransformer/ImageTagTransformer.go

```diff
@@ -151,7 +151,8 @@ func (p *plugin) findContainers(obj map[string]interface{}) error {
 func isImageMatched(s, t string) bool {
 	// Tag values are limited to [a-zA-Z0-9_.-].
 	pattern, _ := regexp.Compile("^" + t + "(:[a-zA-Z0-9_.-]*)?$")
-	return pattern.MatchString(s)
+	patternWithDigest, _ := regexp.Compile("^" + t + "@sha256(:[a-zA-Z0-9_.-]*)?$")
+	return pattern.MatchString(s) || patternWithDigest.MatchString(s)
```

Comment:
You can combine those two regexp into one. Something like this 
```
(@sha256)?(:[a-zA-Z0-9_.-]*)?$
```

Reply:
Thanks. Just changed the code according to comment